data/sql/updates/pending_db_world/rev_1614130170345582600.sql

```diff
@@ -0,0 +1,140 @@
+(3012920, 12, -1672.47, -3263.53, 25.0571, 5.57397, 15000),
+(3012920, 13, -1668.16, -3255.58, 25.73, 2.7269, 0),
+(3012920, 14, -1673.27, -3253.71, 25.7309, 1.5056, 0);
+-- add cleave and berserker stance
+SET @ENTRY := 2606;
+DELETE FROM smart_scripts WHERE entryOrGuid = 2606 AND source_type = 0;
+UPDATE creature_template SET AIName="SmartAI" WHERE entry= @ENTRY;
+INSERT INTO smart_scripts (entryorguid, source_type, id, link, event_type, event_phase_mask, event_chance, event_flags, event_param1, event_param2, event_param3, event_param4, action_type, action_param1, action_param2, action_param3, action_param4, action_param5, action_param6, target_type, target_param1, target_param2, target_param3, target_x, target_y, target_z, target_o, comment) VALUES
+(@ENTRY, 0, 0, 0, 60, 0, 100, 0, 1000, 1000, 240000, 240000, 11, 2458, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, "Every 240 seconds (1s initially) - Self: Cast spell Berserker Stance (2458) on Self"),
+(@ENTRY, 0, 1, 0, 0, 0, 100, 0, 7800, 10000, 11100, 24200, 11, 17207, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, "Every 11.1 - 24.2 seconds (7.8 - 10s initially)  - Self: Cast spell Whirlwind (17207) on Victim"),
+(@ENTRY, 0, 2, 0, 0, 0, 100, 0, 7000, 13000, 7000, 23000, 11, 15496, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, "Every 7 - 23 seconds (7 - 13s initially)  - Self: Cast spell Cleave (15496) on Victim"),
+(@ENTRY, 0, 3, 0, 12, 0, 100, 1, 0, 30, 0, 0, 11, 7160, 1, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, "When victim health between 0 - 30 (check every 0 - 0) - Self: Cast spell Execute (7160) on Victim (flags: interrupt previous)"),
+(@ENTRY, 0, 4, 0, 2, 0, 100, 1, 0, 15, 0, 0, 25, 1, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, "When health between 0 and 15 (check every 0 - 0 ms) - Self: Flee for assist");
+
+-- dustwallow marsh
+-- drogoth the roamer
+INSERT INTO `creature` (`guid`, `id`, `map`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `wander_distance`, `MovementType`) VALUES
+(300769, 14231, 1, -2608.36, -3011.72, 29.4168, 0.0195422, 27000, 10, 1),
+(300770, 14231, 1, -2799.72, -3129.45, 28.7032, 4.41743, 27000, 5, 1);
+INSERT INTO `pool_creature` (`guid`, `pool_entry`, `chance`, `description`) VALUES (300769, 42939, 0, 'Dustwallow Marsh: Drogoth the Roamer');
+INSERT INTO `pool_creature` (`guid`, `pool_entry`, `chance`, `description`) VALUES (300770, 42939, 0, 'Dustwallow Marsh: Drogoth the Roamer');
+-- ripscale
+INSERT INTO `creature` (`guid`, `id`, `map`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `wander_distance`, `MovementType`) VALUES
+(300775, 14233, 1, -3969.57, -3182.07, 28.8974, 3.08685, 38000, 10, 1),
+(300776, 14233, 1, -4040.93, -3557.63, 28.5225, 5.65789, 38000, 10, 1);
+INSERT INTO `pool_creature` (`guid`, `pool_entry`, `chance`, `description`) VALUES (300775, 43157, 0, 'Dustwallow Marsh: Ripscale');
+INSERT INTO `pool_creature` (`guid`, `pool_entry`, `chance`, `description`) VALUES (300776, 43157, 0, 'Dustwallow Marsh: Ripscale');
```

Comment:
DELETE FROM `pool_creature` WHERE `guid` IN (300769,300770);
INSERT INTO `pool_creature` (`guid`, `pool_entry`, `chance`, `description`) VALUES
(300769, 42939, 0, 'Dustwallow Marsh: Drogoth the Roamer'),
(300770, 42939, 0, 'Dustwallow Marsh: Drogoth the Roamer');